<launch>

  <node pkg="gmapping" type="slam_gmapping" name="slam" output="screen" >
    <remap from="scan" to="scan_raw" />
    <?ignore
    <param name="odom_frame" value="/mobile_base_controller/odom" />
    ?>
    <param name="base_frame" value="base_footprint" />
    <param name="xmin" value="0" />
    <param name="ymin" value="0" />
    <param name="xmax" value="10.0" />
    <param name="ymax" value="10.0" />
  </node>

  <node name="det_nav_goal" pkg="navigation_goal" type="determine_navigation_goal.py" output="screen" />

  <node name="set_nav_goal" pkg="navigation_goal" type="set_navigation_goal.py" output="screen" />

</launch>